core/jvm/src/test/scala/scalaz/zio/AbstractRTSSpec.scala

```diff
@@ -0,0 +1,13 @@
+package scalaz.zio
+
+import org.specs2.Specification
+
+trait AbstractRTSSpec extends Specification with RTS {
```

Comment:
I think error handling is very specific to every case, don't think too many people will reuse this. I would rename it to `MuteRTS` and move it to the tests sources. I did that for my tests.